tests/verilog/atom_type_signedness.ys

```diff
@@ -0,0 +1,19 @@
+read_verilog -dump_ast1 -dump_ast2 -sv <<EOT
+module dut();
+
+enum integer  { uInteger  = -10 } a;
+enum int      { uInt      = -11 } b;
+enum shortint { uShortInt = -12 } c;
+enum byte     { uByte     = -13 } d;
+
+always_comb begin
+  assert($signed(-10) == uInteger);
+  assert($signed(-11) == uInt);
+  assert($signed(-12) == uShortInt);
+  assert($signed(-13) == uByte);
```

Comment:
Minor nit: Is `-13`, etc. already signed? I'd be surprised if this test required these explicit casts.

Reply:
True, it is not needed, I removed explicite cast.